workers/unity/Packages/com.improbable.gdk.tools/GenerateFiles.cs

```diff
@@ -0,0 +1,37 @@
+using System.IO;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Tools
+{
+    public class GenerateFiles
+    {
+        private static readonly string
+            ResourcesDir = Path.GetFullPath(Path.Combine(Application.dataPath, "Resources"));
+
+        [MenuItem("SpatialOS/Generate Dev Authentication Token", false, MenuPriorities.GenerateDevAuthToken)]
+        private static void GenerateDevAuthToken()
+        {
+            var receivedMessage = string.Empty;
+            RedirectedProcess
+                .Command(Common.SpatialBinary)
+                .WithArgs("project", "auth", "dev-auth-token", "create", "--description", "\"Dev Auth Token\"")
+                .InDirectory(Common.SpatialProjectRootDir)
+                .AddOutputProcessing((message) => receivedMessage += message)
+                .Run();
+            var regex = new Regex(@"""id\\"": \\""(.*?)\\""");
+            var devAuthToken = regex.Match(receivedMessage).Groups[1].Value;
```

Comment:
If the regex match fails - will this throw in the indexing operation?

Reply:
it actually won't, but yes. need to add some error handling